src/antsibull_changelog/cli.py

```diff
@@ -196,7 +196,7 @@ def create_argparser(program_name: str) -> argparse.ArgumentParser:
     )
     lint_changelog_yaml_parser.add_argument(
         "--strict",
-        type=parse_boolean_arg,
+        action="store_true",
```

Comment:
Should this be BooleanOptionalArg like we use in the other antsibull tools instead?

Reply:
I guess we can use that in the future to also support `--no-strict`, but then we first have to check again what was broken with `BooleanOptionalArg` since antsibull-changelog does not depend on antsibull-core, which has a fixed version for some 3.9.x and 3.10.x Python versions.

Reply:
I think that was only a problem when `default` wasn't present - which is not a problem for `--strict`, but is one for the other two options. Maybe we should at the same time drop support for Python < 3.10.3, since it's only broken for 3.9.x with x < 11 and 3.10.x with x < 3.